libsrc/plus4/Makefile.inc

```diff
@@ -0,0 +1,3 @@
+../target/plus4/drv/tgi/plus4-lo.tgi:   \
+                ../libwrk/plus4/kplot.o \
+                ../libwrk/plus4/kbsout.o
```

Comment:
There seems to be a misunderstanding:

There are two mutually exclusive ways to get C library functionality into a driver: **Either** duplicate it in the driver **or** reference it.

* This commit has the title 'Prepare Plus/4 libref.' but actually it prepares duplication.
* As discussed duplication will never place the duplicated code in LOWCODE.